README.md

```diff
@@ -12,3 +12,53 @@ Grafana or the like.
 This is a [Haskell stack tools](https://docs.haskellstack.org/) based
 project.  Given a working `stack`, one should be able to execute
 `stack build` which will fetch all of the dependencies and build.
+
+After building, you will have several executables.
+
+teslauth: Authenticates to Tesla's API (with your credentials) and
+stores them in a local database.  _For obvious reasons_ treat this
```

Comment:
Is it worth pointing out that the password is *not* stored?

Reply:
heh, well, I actually didn't read that deeply.  so, yes.